tvision/include/tv/tvconfig.h

```diff
@@ -33,17 +33,7 @@ const int maxReplaceStrLen = 80;
 #include <tv/intl.h>
 
 #ifndef pid_t
-   #ifdef HAVE_64BITS_POINTERS
-      #ifdef TVOS_Win32
-        #define pid_t __int64
-      #else
-        #ifdef TVOSf_Linux
-          #define pid_t int64_t
-        #endif
-      #endif
-   #else
-     #define pid_t int
-   #endif
+  #define pid_t unsigned int
```

Comment:
Why not just use:

```c++
#ifndef pid_t
  #if defined(TVCompf_MinGW) && defined(HAVE_64BITS_POINTERS)
    #define pid_t DWORD
  #else          
    #define pid_t int
  #endif
#endif
```

Instead of `DWORD` you could use `__int64`, like [here](https://mingw-w64-public.narkive.com/zGNfd7ET/patch-fix-data-types-pid-t-and-pid-t-for-64-bit-windows)

Reply:
Please see the final change in green.  After discovering that Windows defines a PID to be a DWORD, which is an unsigned int, I got rid of the complicated logic  and replaced it with a single define for all platforms.

` #define pid_t unsigned int`